server/server.js

```diff
@@ -342,6 +346,151 @@ app.post('/message', jsonBodyParser, async (req, res) => {
     }
 });
 
+app.post('/heartbeat', jsonBodyParser, async (req, res) => {
+    log(`got a heartbeat from ${req.body.system_id}, flic_last_seen_secs is ${req.body.flic_last_seen_secs}, flic_last_ping_secs is ${req.body.flic_last_ping_secs}`)
+    let heartbeat = await db.getHeartbeatWithSystemId(req.body.system_id)
+    heartbeat.flicLastSeenTime = moment().subtract(req.body.flic_last_seen_secs, 'seconds').toISOString()
+    heartbeat.flicLastPingTime = moment().subtract(req.body.flic_last_ping_secs, 'seconds').toISOString()
+    heartbeat.heartbeatLastSeenTime = moment().toISOString()
+    await db.saveHeartbeat(heartbeat)
+    res.status(200).send()
+})
+
+
+app.post('/rename_system', jsonBodyParser, async (req, res) => {
```

Comment:
I suggest that we put these heartbeat-related calls under some parent path. Like
`/hearrbeat/rename_system` 
`/heartbeat/hide_system`
`/heartbeat/mute_system`
etc
I think that it would make it more clear what these routes are related to and that there won't be any confusion about ones like `rename_system` and someone thinking that they were renaming an installation or who knows what, lol.

What do you think?

If you make this change, then the `utils.py` will also need to be updated.